curvefs/src/metaserver/partition.cpp

```diff
@@ -404,10 +393,9 @@ uint32_t Partition::GetPartitionId() const {
 }
 
 PartitionInfo Partition::GetPartitionInfo() {
-    auto partitionInfo = partitionInfo_;
-    partitionInfo.set_inodenum(GetInodeNum());
-    partitionInfo.set_dentrynum(GetDentryNum());
-    return partitionInfo;
+    partitionInfo_.set_inodenum(GetInodeNum());
```

Comment:
it seems the caller will GetInodeNum and GetDentryNum after get the partition.

https://github.com/opencurve/curve/blob/feba0744a602a39fddd9addcaf51f96937b793cd/curvefs/src/metaserver/metastore_fstream.cpp#L250